### ISSUE TITLE
`pipe storage cp` windows with the GCS - sslv3 error

### ISSUE THREAD

**Issue by sidoruka** [epam/cloud-pipeline#1268]
**Describe the bug**
We observe the issue with copying data to/from the GCS using the Windows workstation:
```
C:\pipe>pipe.exe storage cp file.txt gs://cloud-pipeline-gcs/file.txt
Error: HTTPSConnectionPool(host='www.googleapis.com', port=443): Max retries exc
eeded with url: /storage/v1/b/cloud-pipeline-gcs/o?projection=noAcl&prefix=file.txt
&delimiter=%2F&versions=False (Caused by SSLError(SSLError("bad handshake: Er
ror([('SSL routines', 'ssl3_read_bytes', 'tlsv1 alert protocol version')],)",),)
```

**Additional context**
The windows `Internet Options` allow the following protocols:
```
SSL 3
TLS 1.0
SSL 1.1
SSL 1.2
```
It might be that the `pipe` uses these settings and the GCP rejects SSLv3 requests

**Comment by sidoruka** [comment]
Fix shall be backported to `release/0.16`, if the issue is caused by the `pipe`

**Comment by mzueva** [comment]
cherry pick 832b6a863bc6817396eeda1bfc1c80a8aa571ba4
